README.md

```diff
@@ -1,11 +1,30 @@
-# eSchool
-Microservice based eSchool Solution
+# eSchool - Microservice and Containers based open source School Administration Software
 
 ## Linux Build Status for 'master' branch
 
 |                                                                                                    Identity API                                                                                                    |
 | :----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------: |
 | [![Build Status](https://dev.azure.com/OpenCodeFoundation/eSchool/_apis/build/status/identity?branchName=master)](https://dev.azure.com/OpenCodeFoundation/eSchool/_build/latest?definitionId=1&branchName=master) |
 
+## Tech & Tools
+- .NET Core
+- Docker
+- Orchestrators: Kubernetes
+- Visual Studio
+- MongoDB
+- SQL Server
+- Azure DevOps
+- Redis
+- RabbitMQ
 
+## Updated for .NET Core 3.0 preview 3 "wave" of technologies
+eSchool is updated to .NET Core 3.0 preview 3 "wave" of technologies. Not just compilation but also new recommended code in EF Core, ASP.NET Core, and other new related versions.
 
+> ### DISCLAIMER
+> **IMPORTANT:** The current state of this application is **BETA**, because we are constantly evolving towards newly released technologies. Therefore, many areas could be improved and change significantly while refactoring the current code and implementing new features. Feedback with improvements and pull requests from the community will be highly appreciated and accepted.
+>
+> However, this application should be considered as an "School administration or management software" at all. The implemented business domain might be ideal from an School administration or management point of view. 
```

Comment:
I am having trouble understanding what you are trying to tell here. I think we can not just copy-paste content from `eShopOnContainers` here because we are trying to develop production grade real application, not reference application like `eShopOnContainers`.